openlibrary/plugins/upstream/utils.py

```diff
@@ -854,6 +855,56 @@ def today():
     return datetime.datetime.today()
 
 
+class HTMLTagRemover(HTMLParser):
+
+    def __init__(self):
+        super().__init__()
+        self.data = []
+
+    def handle_data(self, data):
+        self.data.append(data)
+
+
+@public
+def reformat_html(html_str, max_length=None, root_element=None):
```

Comment:
The `root_element` needlessly complicates things, and can be done in the template itself.  Will remove this later today.